ray-operator/controllers/ray/raycluster_controller_unit_test.go

```diff
@@ -3571,78 +3573,102 @@ func Test_ReconcileManagedBy(t *testing.T) {
 	}
 }
 
-func TestCollectRayClusterMetrics(t *testing.T) {
-	tests := []struct {
-		creationTime     time.Time
+func TestEmitRayClusterProvisionedDuration(t *testing.T) {
+	clusterName := "test-ray-cluster"
+	clusterNamespace := "default"
+
+	// Creation time 5 minutes ago to simulate cluster runtime
+	creationTime := time.Now().Add(-5 * time.Minute)
+
+	testCases := []struct {
 		name             string
-		clusterName      string
-		clusterNamespace string
-		newClusterStatus rayv1.RayClusterStatus
-		oldClusterStatus rayv1.RayClusterStatus
+		originStatus     rayv1.RayClusterStatus
```

Comment:
I’ve seldom seen variables named `originStatus`. `oldStatus`, `originalStatus`, and `initialStatus` are more common.

```suggestion
		oldStatus     rayv1.RayClusterStatus
```

Reply:
Addressed in https://github.com/ray-project/kuberay/pull/3497/commits/b07967fb140d08e8933c2eddd91c74e4fb7b3763